content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+
+
+## cloudify.gcp.nodes.DNSTXTRecord
+**Derived From:** cloudify.gcp.nodes.DNSRecord
+
+`TXT` type DNSRecord
+
+
+
+**Properties:**
+
+
+  * `type`
+
+
+    *default:* TXT
+
+
+
+
+
+## cloudify.gcp.nodes.DNSZone
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A Cloud DNS zone.
+
+Represents a particular DNS domain which you wish to manage through Google Cloud DNS.
+DNS nameservers can vary between different DNSZones. In order to find the correct nameserver entries for your domain, use the `nameServers` attribute from the created zone.
+
+
+
+**Properties:**
+
+
+  * `dns_name`
+    (fully qualified) domain name of the zone. Defaults to the instance ID.
+
+    *type:* string
+    *default:*
+  * `additional_settings`
+    Additional settings
+
+    *default:* {}
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true)  or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    (internal) name of the zone. Defaults to the instance ID.
+
+    *type:* string
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+
+
+
+### Example
+
+{{< gsHighlight  yaml  >}}
+
+my_zone:
+  type: cloudify.gcp.nodes.DNSZone
+  properties:
+    dns_name: getcloudify.org.
+
+{{< /gsHighlight >}}
+
+The `dns_name` supplied must be a fully-qualified domain name with the trailing dot. The output attributes (`runtime_properties`) will include a key `nameServers` which contains the list of nameservers that should be supplied as nameservers with the domain registrar.
+
+
+
+## cloudify.gcp.nodes.ExternalIP
+**Derived From:** [cloudify.nodes.VirtualIP]({{< relref "blueprints/built-in-types.md" >}})
+
+Use this together with the `cloudify.gcp.relationships.instance_connected_to_ip` if you want the Instance to have an ephemeral external IP
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `ip_address`
+    Address of this external IP. This should be address of already existing, unattached static IP. It will be used only if "use_external_resource" is set to true.
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource. If set to true, this node will be static IP, otherwise ephemeral IP.
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.FirewallRule
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP FirewallRule.
+
+This describes allowed traffic directed to either the whole of the specified network, or to Instances specified by matching tags.
+
+
+
+**Properties:**
+
+
+  * `sources`
+    List of CIDR formatted ranges and instance tags which
```

Comment:
**The** ~~L~~**l**ist of CIDR formatted ranges and instance tags ~~which~~ **that**